xbmc/Application.cpp

```diff
@@ -1081,10 +1081,30 @@ bool CApplication::InitDirectoriesLinux()
   else
     userHome = "/root";
 
+  /* Get XDG paths.
+   * Though a slash is expected at the end anyway, let's cover ourselves, just in case. */
+  std::string xdgDataPath;
+  if (getenv("XDG_DATA_HOME"))
+  {
+    xdgDataPath = getenv("XDG_DATA_HOME");
+    URIUtils::AddSlashAtEnd(xdgDataPath);
```

Comment:
whatever you think is best, as long the old behaviour is there. of course if the team agrees to keep it. if not - fine. to be clear, I am not pushing for changes you dont lke. there is nothing wrong in following standards.